docs/supported-devices.md

```
@@ -151,6 +151,14 @@ Successful connection requires firmware version **A0-MEV-V2.0.9** or newer.
 
 The "Charger idle sampling interval" is not supported. Set this to **0** to avoid a "ClockAlignedDataInterval is read-only" warning.
 
+## [Ocular LTE Plus v3](https://evse.com.au/) (tested: reports as BS-EV22, firmware 448.3251.0Q03197)
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)